docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -599,3 +599,33 @@ Then `${context-width}` is > `0`
 Then `${context-x-coordinate}` is > `0`
 Then `${context-y-coordinate}` is > `0`
 ----
+
+
+=== Gets a list of elements by locator and check that it is sorted by text with using sortingOrder by element text
```

Comment:
`it is sorted by text with using sortingOrder by element text` - this part should be reworked